lib/main.js

```diff
@@ -1,5 +1,6 @@
 var _ = require('lodash');
-var css = require('css');
+var postcss = require('postcss');
```

Comment:
With bigger communities they should have less bugs and handle more edge-cases, which is crucial for us because of our mmmmega hax.